avocado/core/test.py

```diff
@@ -469,6 +469,9 @@ def _run_avocado(self):
         except exceptions.TestSkipError as details:
             stacktrace.log_exc_info(sys.exc_info(), logger='avocado.test')
             raise exceptions.TestSkipError(details)
+        except exceptions.TestSkip as details:
```

Comment:
I understand the difference between `TestSkipError`, used on `self.skip()`, and this new `TestSkip` exception. I wonder, though, if we should name them more clearly (involves renaming `TestSkipError`).

I haven't settled my mind even on a suggestion, so, for now, this is just a braindump and opportunity for brainstorm.